main/frame.cjs

```diff
@@ -0,0 +1,63 @@
+class Frame {
+  constructor() {
+    this.score = 0;
+  }
+
+  addRoll(points) {
+    this.#validateRoll(points);
+    this.roll1 || this.roll1 === 0 ? (this.roll2 = points) : (this.roll1 = points);
```

Comment:
I'd suggest expanding this rather than using a ternary, to improve readability (ternary conditions are fine when used as evaluated expressions, but less obvious to read when the different branches of the conditions contain instructions, such as assigning a variable) - and then wrap it in a private method as well. Maybe something like:
```js
#updateRoll(points) {
  if (this.roll1 || this.roll1 === 0) {
    this.roll2 = points
  } else {
    this.roll1 = points
  }
}
```